squash-honeycomb.ipynb

```diff
@@ -0,0 +1,381 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# [Honeycomb](https://www.honeycomb.io/)\n",
+    "Fast exploration over high cardinality data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Getting started\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Check the [quick start tutorial](https://docs.honeycomb.io/quick-start) to learn about Honeycomb UI capabilities. \n",
+    "Then join the LSST SQuaRE team using [this invitation](https://ui.honeycomb.io/join_team/lsst-square), the datasets created in this notebook are shared with the team.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## What is observability?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Honeycomb [observability manifesto](https://www.honeycomb.io/blog/2018/03/observability-a-manifesto/) is a good place to start. Also check the introduction to observability [here](https://docs.honeycomb.io/thinking-about-observability/intro-to-observability/)\n",
+    "\n",
+    "NOTE: KPMs are high level metrics, they are not the kind of 'value' that one would want for observability. However,\n",
+    "in this notebook we'll send KPMs to Honeycomb just because KPMs are the only data that we have in SQuaSH so far. That's still useful to illustrate the use of `libhoney` and `beeline-python` integrations.\n",
+    "\n",
+    "In my opinion, the benefit of this tool will become clearer by sending more *context data* from our DM pipeline tasks, and ultimately as an analysis tool for the Engineer Facility Database (EFD). \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Events and Datasets\n",
+    "Honeycomb data is a series of **Events**, each of which represents something in your environment worth tracking. When you send events to Honeycomb, you collect related or recurring events under a single **Dataset**.\n",
+    "\n",
+    "An **Event** can be anything:\n",
+    "\n",
+    "- an HTTP request to your app\n",
+    "- An SQL query\n",
+    "- A job is submitted to a queue\n",
+    "- An execution of a LSST DM pipeline task\n",
+    "- A detection of particular astronomical object \n",
+    "\n",
+    "**Datasets** are used to partition your data into separate and queryable sets. See [best practices for defining datasets](https://docs.honeycomb.io/getting-data-in/datasets/best-practices/) in Honeycomb. \n",
+    "\n",
+    "Let's start by identifying a `validate_drp` run as our event and creating a new dataset for the metrics measured by each run. \n",
```

Comment:
Don't you mean each run is an `Event`, not dataset?

Reply:
Yes, I mean each run is an `Event`. Rephrased like this to avoid confusion:

Let's start by identifying a `validate_drp` run as the `Event` containing the metrics measured by each run.